### PR TITLE
Correction warning and add display option

### PR DESCRIPTION
**Pull request by Exspiravit**
Hi bro,
I add the display option to make them appear and disappear from both the menu and the front page.
Fixed some warnings for using class instead of class name.
I took the liberty of adding the codersrank library so that you can easily add your profile to the portfolio.

### CODE REVIEW

**Comment by Exspiravit** [comment]
@saadpasta bro I do not know if I had to join the readme or you do it when you think that the collaboration has been valuable so in case the flies do not add me hehe: 3

luck and thanks for publishing this project has helped me a lot.

File: package.json
Comment (saadpasta) [inline package.json:1]: Please remove package.json from your PR

File: src/components/button/Button.js
Comment (saadpasta) [inline src/components/button/Button.js:6]: Why did you remove className from div? Please undo this.

File: src/containers/achievement/Achievement.js
Comment (saadpasta) [inline src/containers/achievement/Achievement.js:11]: If this function is not used anymore please remove.

File: src/containers/greeting/Greeting.js
Comment (saadpasta) [inline src/containers/greeting/Greeting.js:19]: Please rename to className

**Comment by saadpasta** [comment]
@Exspiravit what's the motivation behind adding codersrank

**Comment by kartik918** [comment]
@saadpasta are you sure we need this? Seems like a pretty specific to one person's needs feature. Only helps those with CodeRank profiles.

**Comment by saadpasta** [comment]
@kartik918 I am also not up for CodeRank but other changes are good.

**Comment by kartik918** [comment]
What else is new in this @saadpasta, I'm having trouble understanding sorry.

**Comment by saadpasta** [comment]
Other than coderRank many warning has been removed and display option for every component is added but we need to test that

**Comment by Exspiravit** [comment]
mmmm I thought it would be practical to add the option for people who had a profile in codesrank to enter the component, I do not really have any motivation beyond that, since I do not work with them, if I have a profile on their page but nothing else .
For the rest, some of the corrections you mentioned made me an error when evaluating the code in codacy, however I will make the changes you mention and I will eliminate the import of codesrank since in case you are right it is perhaps very personal with a service and it would be better to keep the profile as generic as possible.

File: package.json
Comment (Exspiravit) [reply package.json]: yeah sorry i forgot remove my personal changes when divided branches.

**Comment by Exspiravit** [comment]
Sorry for the second deployment, i forgot remove codesrank library.

**Comment by saadpasta** [comment]
@Exspiravit Please remove package.json and yarn.lock from your PR

**Comment by kartik918** [comment]
Remember to squash the PR when merging, also great work @Exspiravit !

**Comment by naveen521kk** [comment]
@saadpasta lets merge this but after the dark mode gets merged. Or it may cause some issues.

**Comment by Exspiravit** [comment]
its ready to merge :3

**Comment by saadpasta** [comment]
> its ready to merge :3

We will merge this after #92

File: src/components/header/Header.js
Comment (Ergouzii) [inline src/components/header/Header.js:41]: Hey nice catch on these conditional rendering! 

What about this shorter version of code:
```
{openSource.display &&
   <li>
      <a href="#opensource">Open Source</a>
    </li>
}
```

Comment (Exspiravit) [reply src/components/header/Header.js:41]: mmmm i don't really think about that but its a good point maybe i change when resolve merge conflicts. :+1:

Comment (saadpasta) [reply src/components/header/Header.js:41]: @Ergouzii we have merged #92 can you resolve conflicts now

Comment (Ergouzii) [reply src/components/header/Header.js:41]: @saadpasta Hey I'm not the author of this PR so I can't resolve the conflicts on my side.

Comment (kartikcho) [reply src/components/header/Header.js:41]: Ah, I think he meant to ping @exspiravit.

**Comment by saadpasta** [comment]
@Exspiravit  Please resolve conflicts so we can merge this PR ✌

**Comment by kartikcho** [comment]
Off topic but I'm getting messages to add `Hacktoberfest` topic to the repo so people who contributed for it can get their prizes. @saadpasta

**Comment by saadpasta** [comment]
> Off topic but I'm getting messages to add `Hacktoberfest` topic to the repo so people who contributed for it can get their prizes. @saadpasta

Sure. You can add it and also we need to add `hacktoberfest-accepted` on accepted PR's

**Comment by kartikcho** [comment]
> Sure. You can add it and also we need to add `hacktoberfest-accepted` on accepted PR's

I think we only need to do one thing according to their [updated rules](https://hacktoberfest.digitalocean.com/hacktoberfest-update). Also, I can't access repo settings so you'll have to add the repo topic yourself 😅.

**Comment by kartikcho** [comment]
Hey @Exspiravit , could you resolve this soon as it is becoming a blocker for other PRs. Thanks

**Comment by Exspiravit** [comment]
> Hey @Exspiravit , could you resolve this soon as it is becoming a blocker for other PRs. Thanks

yea i resolve this today or tomorow.

**Comment by Exspiravit** [comment]
hey bros @saadpasta and @kartikcho i finaly have time to merge this and correction some issues, but need authorization for test in vercel.
Sorry for delay i have some issues with my internet connection  :disappointed: 
Whatever i whant to continue colaborate in this proyect more :fox_face: thank you for your patient.

File: .all-contributorsrc
Comment (saadpasta) [inline .all-contributorsrc:286]: Please remove this we would add you as a contributor when the PR get's merged

File: package.json
Comment (saadpasta) [inline package.json:9]: Why you have added axios ?

File: .all-contributorsrc
Comment (Exspiravit) [reply .all-contributorsrc:286]: ok i remove it :+1:

File: package.json
Comment (Exspiravit) [reply package.json:9]: sorry error when merge with my personal fork.

File: src/portfolio.js
Comment (saadpasta) [inline src/portfolio.js:194]: set `showGithubProfile `to true and add `githubUserName` **saadpasta**

File: src/components/header/Header.js
Comment (saadpasta) [inline src/components/header/Header.js:36]: Revert to `Work Experience`  and please make this check for all the components that only those with true show's on the header

**Review by saadpasta — COMMENTED** [review]
Please `remove package-lock.json`

**Comment by Exspiravit** [comment]
> Please remove `package-lock.json`

ready

**Comment by saadpasta** [comment]
> > Please remove `package-lock.json`
> 
> ready

@Exspiravit Please resolve other comments too. Thanks

File: src/portfolio.js
Comment (Exspiravit) [reply src/portfolio.js]: ready :+1:

File: src/components/header/Header.js
Comment (Exspiravit) [reply src/components/header/Header.js]: ready

File: src/portfolio.js
Comment (saadpasta) [inline src/portfolio.js:14]: @Exspiravit Look's like you didn't get my point. 
Remove Name of the Section from the header if the section is displayed false.
For E.g 
Is Education Section is `display: false` **Education**  key should not appear in the header.
Please remove this key. 

Sorry for not clearing myself before

File: src/portfolio.js
Comment (saadpasta) [inline src/portfolio.js:193]: Put `true` in string

**Comment by kartikcho** [comment]
Hey again @Exspiravit, pinging you to resolve the remaining tasks from the review to get this merged.

**Comment by saadpasta** [comment]
Look's like @Exspiravit is not active and we have to fix this on our own. I fix that and will push it soon.

**Comment by naveen521kk** [comment]
@saadpasta Is it done?

**Comment by saadpasta** [comment]
@kartikcho @naveen521kk Changes done please review. 
Sorry for being late 😞

File: src/containers/contact/Contact.js
Comment (kartikcho) [inline src/containers/contact/Contact.js:10]: We don't have this property in `contactInfo`, I don't think we should extend to include this either because contact information is pretty essential for a portfolio.

**Review by kartikcho — CHANGES_REQUESTED** [review]
Everything looks great apart from a few minor changes. Thanks for picking this up and refining the PR!

**Comment by saadpasta** [comment]
@kartikcho changes done. Please review ✌

**Review by kartikcho — APPROVED** [review]
Everything looks good, LGTM!

**Pull request merged**
